cmd/todoapp.go

```diff
@@ -19,6 +24,7 @@ func main() {
 	updateText := flag.String("update-text", "", "the new description for the item")
 	updateStatus := flag.String("update-status", "", "the new status for the item (not started, started, completed)")
 	deleteID := flag.Int("delete-id", 0, "the ID of the item you want to delete")
+	serveAPI := flag.Bool("start-server", false, "Start HTTP API server")
```

Comment:
Not something you must change, but I personally prefer:
```suggestion
        var serveAPI bool 
	flag.BoolVar(&serveAPI, "start-server", false, "Start HTTP API server")
```